src/backend/src/utils/change-requests.utils.ts

```diff
@@ -27,33 +27,108 @@ export const sendSlackChangeRequestNotification = async (
   message: string,
   crId: number,
   budgetImpact?: number
-) => {
-  if (process.env.NODE_ENV !== 'production') return; // don't send msgs unless in prod
-  const msgs = [];
+): Promise<{ channelId: string; ts: string }[]> => {
```

Comment:
add some documentation to this method as well and make sure to describe the information that it returns